CMakeLists.txt

```diff
@@ -576,7 +582,7 @@ endif()
 if(${HIP_CONTIGUOUS_MEMORY})
   target_compile_definitions(rccl PRIVATE HIP_CONTIGUOUS_MEMORY)
 endif()
-if(${hipcc_version_string} VERSION_GREATER_EQUAL "5.7.31920")
+if(${compiler_version_string} VERSION_GREATER_EQUAL "5.7.31920")
```

Comment:
@lawruble13 this one seems to be related to hipcc version... is there an equivalent for amdclang version?